Gemfile.lock

```diff
@@ -51,112 +51,637 @@ GEM
+      aws-sigv4 (~> 1.0)
+    aws-sdk-sagemakerruntime (1.0.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-serverlessapplicationrepository (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-servicecatalog (1.4.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-servicediscovery (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-ses (1.6.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-shield (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-simpledb (1.0.1)
+      aws-sdk-core (~> 3)
+      aws-sigv2 (~> 1.0)
+    aws-sdk-sms (1.0.1)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-snowball (1.2.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-sns (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-sqs (1.3.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-ssm (1.8.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-states (1.2.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-storagegateway (1.3.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-support (1.0.1)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-swf (1.0.1)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-transcribeservice (1.0.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-translate (1.0.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-waf (1.4.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-wafregional (1.4.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-workdocs (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-workmail (1.0.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-workspaces (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-xray (1.1.0)
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sigv2 (1.0.1)
+    aws-sigv4 (1.0.2)
+    backports (3.11.1)
+    bunny (2.9.2)
+      amq-protocol (~> 2.3.0)
     coder (0.4.0)
-    coderay (1.1.1)
+    coderay (1.1.2)
     concurrent-ruby (1.0.5)
     connection_pool (2.2.1)
     dalli (2.7.6)
     diff-lcs (1.3)
     docile (1.1.5)
-    faraday (0.12.2)
+    faraday (0.14.0)
       multipart-post (>= 1.2, < 3)
     hashr (2.0.1)
     hitimes (1.2.6)
     httpclient (2.8.3)
-    i18n (0.8.6)
+    i18n (0.9.5)
+      concurrent-ruby (~> 1.0)
     jmespath (1.3.1)
     json (2.1.0)
-    jwt (1.5.6)
-    method_source (0.8.2)
-    minitest (5.10.3)
-    multi_json (1.12.1)
+    jwt (2.1.0)
+    method_source (0.9.0)
+    minitest (5.11.3)
+    multi_json (1.13.1)
     multipart-post (2.0.0)
-    oj (3.3.2)
-    parallel (1.11.2)
-    parser (2.4.0.0)
-      ast (~> 2.2)
-    pg (0.21.0)
+    parallel (1.12.1)
+    parser (2.5.0.2)
+      ast (~> 2.4.0)
+    pg (1.0.0)
```

Comment:
worth watching out for, since there were some issues with this in other apps iirc. may have been activerecord related though.